demo/tests/test_end_point.py

```diff
@@ -652,6 +800,91 @@ def test_create_form_with_presets_invalid_argument(self):
             'Preset (confirmation) argument is using an undefined field (testField2)'  # noqa
         )
 
+    def test_create_form_conditions(self):
```

Comment:
I would be in favor of a few comments that explain the different steps of this test